### ISSUE TITLE
[MYSQL] "datetime" datatype with default value doesn't export properly

### ISSUE THREAD

**Issue by shubham-yb** [yugabyte/yb-voyager#489]
If we create a `datetime` column which has a `default` value, the export does not go through as expected and thus we get syntax error while importing.

Consider this schema:

`create table m (date_status_changed datetime DEFAULT (curtime()));`

This gets exported as:

```
CREATE TABLE m (
	date_status_changed timestamp without time zone DEFAULT CURRENT_TIMESTAMP::(0)::timestamp without time zone(0)
) ;
```
 which is syntactically incorrect.

**Comment by sanyamsinghal** [comment]
@shubham-yb  I just tried this: 
```
CREATE TABLE m (
	date_status_changed timestamp without time zone DEFAULT CURRENT_TIMESTAMP
) ;
```
and it worked

Can you explain what does this part `::(0)::timestamp without time zone(0)` might be trying to do here.

**Comment by shubham-yb** [comment]
@sanyamsinghal I don't know why exactly the entire thing is pushed like that, but `timezone(0)` is meant to remove the milliseconds part and round it off to whole seconds. And same for `CURRENT_TIMESTAMP`.

Example:



![Image](https://user-images.githubusercontent.com/65844208/196636767-e4e4ccf2-78f0-43e5-b141-aea93e1b8f1f.png)

**Comment by shubham-yb** [comment]
Fixed by Ora2pg in their commit `23e01d5197f2024ff47ba01c15ac95ed4ca6fa5e` and tested.
